### PR TITLE
Void return while SHUTDOWN

### PR DESCRIPTION
**Pull request by sazzad16**
The return type of `shutdown()` should be `void`.
Reply on error can be thrown as an exception.

### CODE REVIEW

**Review by dengliming — DISMISSED** [review]

File: src/main/java/redis/clients/jedis/BinaryJedis.java [hunk 1]
```diff
@@ -3469,19 +3470,18 @@ public Long lastsave() {
...
    * {@link #save() SAVE} and then {@link #quit() QUIT} because other clients may alter the DB data
    * between the two commands.
-   * @return Status code reply on error. On success nothing is returned since the server quits and
-   *         the connection is closed.
+   * @throws JedisException with the status code reply on error. On success nothing is thrown since
+   *         the server quits and the connection is closed.
    */
   @Override
-  public String shutdown() {
+  public void shutdown() throws JedisException {
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/BinaryJedis.java:3477]: @sazzad16 shouldn't we deprecate this method first?

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3477]: @gkorland Why deprecate? We're not removing it.

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3477]: @gkorland Please check #2545

**Comment by sazzad16** [comment]
Resolved by https://github.com/redis/jedis/commit/858c8051301854e8343eef66aa68c6b9dab1c328

**Pull request closed without merging**